packages/augur-ui/config/webpack.common.config.js

```diff
@@ -23,9 +23,12 @@ const PATHS = {
 const AUGUR_ENV = process.env.AUGUR_ENV || process.env.ETHEREUM_NETWORK || 'local';
 const config = buildConfig(AUGUR_ENV);
 
-const gitRevisionPlugin = new GitRevisionPlugin({
-  branch: true
-});
+if(!process.env.CURRENT_COMMITHASH) {
+  const gitRevisionPlugin = new GitRevisionPlugin({
```

Comment:
this needs to be available in a broader scope since it's used later in the script